docs/features/protocols/bitcoin.md

```diff
@@ -0,0 +1,16 @@
+# Bitcoin Protocol
+Trezor Suite, both on web and desktop, can handle the opening of URLs with the `bitcoin:` protocol.
+
+## Behaviour on web
+When opening Suite on web, it will prompt the user to associate the opening of `bitcoin:` URLs with Suite. By accepting, all `bitcoin:` URLs in the browser will open Suite. 
```

Comment:
Just something I noticed, in chrome I allowed the web version (running on sldev) to handle bitcoin links and it always opens https://suite.corp.sldev.cz/?uri=bitcoin... (no path after hostname), maybe it will work in production, but missing /web in the url could also cause a problem?
Another kinda UX issue is that after I allowed the web version later I couldn't figure out how to let the desktop app handle these links. Looks like there is no UI for this in Chrome (user has to delete some file). I am sure this will confuse tons of people. 
Is handling bitcoin URI something rally want to have also in web version? Maybe having this in desktop app would suffice.

Reply:
Ah yes indeed, I need to use the `assetPrefix` (or something similar) to handle that on web. I'm pretty sure there is a UI in Chrome to manage how these protocols behave but it can be confusing if the user selected to always open it with Suite Web and doesn't know how to disable that.

Not sure what we want to do here. Should we support bitcoin urls on Suite Web? If yes, do we want to display the registration message (for handling these urls) all the time or after a user interaction? Looks also that it is impossible to de-register it from the application. cc @prusnak @tsusanka @matejzak

Reply:
I wouldn't do it at all in web. If you want to have a richer set of features just go to the desktop version, which we are trying to "sell" more anyway. Also imagine use case where you register it to for web, then install Suite and then what happens? It stays in web? Goes bananas? Who knows :). So my suggestion is to remove any web-related stuff handling and let's reevaluate later if we need it. @matejzak what do you think?

Reply:
@matejzak raised a very good point that currently `bitcoin:` leads to the old wallet which we would like to override.

Reply:
I'm putting it behind a feature flag, so you guys can flip the switch however you want :slightly_smiling_face:

Reply:
Great. Agreed with @matejzak let's put all web-related behind a feature flag and set it to false for now. We will freeze that tomorrow and we will continue the discussion after the freeze (into the next release). I have created https://github.com/trezor/trezor-suite/issues/3313 to track it.

Reply:
Just discussed with @tsusanka. I believe handling the web as well makes sense since the current "solutions" direct users to my-trezor. Also I can imagine even the hard core users benefiting from paying their coffees on the "lite" web version. That being said, it needs to be investigated/specified thoroughly first. Since it is too close to the freeze, let's leave for the next sprint.

Reply:
What is the result of the investigation? @matejzak  @tsusanka